kong/cli/services/nginx.lua

```diff
@@ -59,11 +59,23 @@ local function get_current_user()
   return IO.os_execute("whoami")
 end
 
+local function get_primary_group(user)
+  return IO.os_execute("id -g -n "..user)
+end
+
+local function is_root()
+  local _, exit_code = IO.os_execute("[[ $EUID -eq 0 ]]")
+  return exit_code == 0
+end
+
 local function prepare_nginx_configuration(configuration, ssl_config)
+
+  local current_user = get_current_user()
+
   -- Extract nginx config from kong config, replace any needed value
   local nginx_config = configuration.nginx
   local nginx_inject = {
-    user = get_current_user(),
+    user = is_root() and "user "..current_user.." "..get_primary_group(current_user)..";" or "",
```

Comment:
```shell
# adduser newuser
# su - newuser -c "kong start"
[INFO] Kong 0.6.1
[INFO] Using configuration: /etc/kong/kong.yml
[INFO] database...........cassandra keyspace=kong ssl=verify=false enabled=false replication_factor=1 contact_points=ec2-52-6-21-95.compute-1.amazonaws.com:9042 replication_strategy=SimpleStrategy timeout=5000 data_centers=
[INFO] dnsmasq............address=127.0.0.1:8053 dnsmasq=true port=8053
[INFO] Auto-generating the default SSL certificate and key...
[INFO] nginx .............admin_api_listen=0.0.0.0:8001 proxy_listen=0.0.0.0:8000 proxy_listen_ssl=0.0.0.0:8443
[INFO] serf ..............-profile=wan -rpc-addr=127.0.0.1:7373 -event-handler=member-join,member-leave,member-failed,member-update,member-reap,user:kong=/usr/local/kong/serf_event.sh -bind=0.0.0.0:7946 -node=7614aadd81f6_0.0.0.0:7946 -log-level=err
[INFO] Trying to auto-join Kong nodes, please wait..
[WARN] Cannot auto-join the cluster because no nodes were found
[OK] Started
```